membership-attribute-service/app/filters/AddGuIdentityHeaders.scala

```diff
@@ -20,11 +20,15 @@ class AddGuIdentityHeaders (implicit val mat: Materializer, ex: ExecutionContext
 
 object AddGuIdentityHeaders {
 
+  //Identity checks for test users by first name
+  def isTestUser(displayName: Option[String]) =
+    displayName.flatMap(_.split(' ').headOption).exists(Config.testUsernames.isValid)
```

Comment:
I know you didn't add this but is there not a firstName field to avoid having to split the displayName? It feels slightly brittle.

Reply:
unfortunately not without doing another call to identity I believe. The `displayName` here is straight out of the cookie. I am double checking before I merge it.

Reply:
yep, the first name is one of the 'Private Fields' in the cookie (unlike `displayName` which is one of the 'Public Fields') 
![image](https://user-images.githubusercontent.com/19289579/49889993-db8fe900-fe3a-11e8-8d09-d1f64ec3780b.png)

Reply:
at least it's only for us, won't break any real users, definitely not worth an additional rest call!
first name is correct based on the behaviour of most of the other subs estate, normally email and last name are ignored.